### PR TITLE
completed issue #72 and #73

### PR DESCRIPTION
**Pull request by shahbhumi53**

### CODE REVIEW

**Comment by shahbhumi53** [comment]
screen shots::
 http://prntscr.com/iyhp07
http://prntscr.com/iyhpgg
http://prntscr.com/iyhprc
http://prntscr.com/iyh99y

File: src/components/EntryList.js [hunk 1]
```diff
@@ -50,7 +50,9 @@ class EntryList extends React.Component {
               style={{
                 ...style,
                 transform: "translateX(-100px)",
-                opacity: "0"
+                opacity: "0",
+                backgroundColor: "blue",
+                height: "500px"
```
Comment (IanMayo) [inline src/components/EntryList.js]: @shahbhumi53  - is it right to fix the height of this component to 500px?  Shouldn't it consume all height?

Note: my connection is to slow to do an npm install, so I can't download & check for myself :-(

**Comment by IanMayo** [comment]
@shahbhumi53  Thanks for the screenshots, nice.

Have inserted question in comment regarding CSS-height.

**Comment by IanMayo** [comment]
@shahbhumi53  - when I type in a course of 123 the box shrinks too far:

screen shot 2018-03-30 at 13 39 50

**Comment by IanMayo** [comment]
Aah, also the case for inline editor:
screen shot 2018-03-30 at 13 42 32

**Pull request merged**